frontend/package.json

```diff
@@ -34,7 +34,9 @@
         "typegen:watch": "cd .. && NODE_OPTIONS=\"--max-old-space-size=16384\" kea-typegen watch --delete --show-ts-errors --use-cache",
         "typegen:write": "cd .. && NODE_OPTIONS=\"--max-old-space-size=16384\" kea-typegen write --delete --show-ts-errors --use-cache",
         "typegen:write:no-cache": "cd .. && NODE_OPTIONS=\"--max-old-space-size=16384\" kea-typegen write --delete --show-ts-errors",
+        "schema:build": "pnpm schema:build:json && pnpm schema:build:validators",
```

Comment:
There's `schema:build` on the root `package.json` already. It tun the json gen + the python gen. You should probably remove this one and just extend that one with the new `validators` one